### PR TITLE
InsynchqPteLtd.Insync version 3.4.0.40973

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16098)

### CODE REVIEW

File: manifests/i/InsynchqPteLtd/Insync/3.4.0.40973/InsynchqPteLtd.Insync.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: InsynchqPteLtd.Insync
```
Comment (ItzLevvie) [inline manifests/i/InsynchqPteLtd/Insync/3.4.0.40973/InsynchqPteLtd.Insync.installer.yaml:3]:
```suggestion
PackageIdentifier: Insynchq.Insync
```

File: manifests/i/InsynchqPteLtd/Insync/3.4.0.40973/InsynchqPteLtd.Insync.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: InsynchqPteLtd.Insync
```
Comment (ItzLevvie) [inline manifests/i/InsynchqPteLtd/Insync/3.4.0.40973/InsynchqPteLtd.Insync.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: Insynchq.Insync
```

File: manifests/i/InsynchqPteLtd/Insync/3.4.0.40973/InsynchqPteLtd.Insync.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: InsynchqPteLtd.Insync
```
Comment (ItzLevvie) [inline manifests/i/InsynchqPteLtd/Insync/3.4.0.40973/InsynchqPteLtd.Insync.yaml:3]:
```suggestion
PackageIdentifier: Insynchq.Insync
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

Make sure you've renamed the folder and all of the file names to match `Insynchq.Insync` using the GitHub Desktop application on Windows.

**Comment by krystofkrticka** [comment]
@ItzLevvie Is it better now?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**